src/main/java/org/pesmypetcare/webservice/thirdpartyservices/adapters/firestore/FirestoreDocumentAdapter.java

```diff
@@ -36,6 +38,17 @@ public WriteBatch batch() {
         return db.batch();
     }
 
+    @Override
+    public void commitBatch(@NonNull WriteBatch batch) throws DatabaseAccessException, DocumentException {
+        try {
+            batch.commit().get();
+        } catch (InterruptedException e) {
+            throw new DatabaseAccessException("write-failed", e.getMessage());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "write-failed" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5492924)